### PR TITLE
Commerce 7543

### PR DESCRIPTION
**Pull request by garofalog**

### CODE REVIEW

**Comment by garofalog** [comment]
ci:test:sf

**Comment by garofalog** [comment]
ci:test:relevant

**Comment by igorbeslic** [comment]
ci:test:relevant

File: modules/apps/batch-planner/batch-planner-web/src/main/resources/META-INF/resources/import/edit_batch_planner_plan.jsp [hunk 1]
```diff
@@ -62,6 +62,32 @@ EditBatchPlannerPlanDisplayContext editBatchPlannerPlanDisplayContext = (EditBat
 								/>
 							</div>
 
+							<div class="alert alert-autofit-stacked alert-indicator-start alert-info alert-inline">
+								<div class="alert-autofit-row autofit-row">
+									<div class="autofit-col autofit-col-expand">
+										<div class="autofit-section">
+											<span class="alert-indicator">
+												<svg class="lexicon-icon lexicon-icon-info-circle" focusable="false" role="presentation">
```
Comment (FabioDiegoMastrorilli) [inline modules/apps/batch-planner/batch-planner-web/src/main/resources/META-INF/resources/import/edit_batch_planner_plan.jsp:70]: `clay:icon` provides a wrapper for icons that makes the code more clean readable. Please use it

File: modules/apps/batch-planner/batch-planner-web/src/main/resources/META-INF/resources/import/edit_batch_planner_plan.jsp [hunk 2]
```diff
@@ -62,6 +62,32 @@ EditBatchPlannerPlanDisplayContext editBatchPlannerPlanDisplayContext = (EditBat
 								/>
 							</div>
 
+							<div class="alert alert-autofit-stacked alert-indicator-start alert-info alert-inline">
```
Comment (FabioDiegoMastrorilli) [inline modules/apps/batch-planner/batch-planner-web/src/main/resources/META-INF/resources/import/edit_batch_planner_plan.jsp:65]: In portal there are occurrences of a `clay:alert` taglib that should do what you're doing here 

[Here's an example ](https://github.com/liferay/liferay-portal/blob/54b94f08e29c1dbcd4d72446fa3906a171ab0843/modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/display/view.jsp#L121)

**Review by FabioDiegoMastrorilli — CHANGES_REQUESTED** [review]

File: modules/apps/batch-planner/batch-planner-web/src/main/resources/META-INF/resources/css/main.scss [hunk 3]
```diff
@@ -0,0 +1,5 @@
+.download-template-link {
+	&.disabled {
+		color: #83b1fb;
```
[the remark this answers is not in the document]
Comment (garofalog) [reply modules/apps/batch-planner/batch-planner-web/src/main/resources/META-INF/resources/css/main.scss:3]: i guess $primary-l2 can fit for disabled text here

**Comment by igorbeslic** [comment]
@garofalog you can close this one... I just overwritten your code with this:
https://github.com/liferay-commerce/liferay-portal/pull/2408

**Pull request closed without merging**